src/Symfony/Component/Messenger/Transport/AmqpExt/Stamp/RoutingKeyStamp.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\AmqpExt\Stamp;
```

Comment:
should be moved to the root Stamp namespace: we should make this stamp transport agnostic (and it should be documented as such, doc PR welcome btw :) )

Reply:
For now, it's Amqp only so let's keep it here. Routing key makes little sense with Redis for example.

Reply:
I don't agree with adding a transport specific stamp. That'd break the purpose of the component and the underlying patterns.
To me a routing key can be used eg to target a specific user. That's totally not amqp specific with the this pov.